src/symbolize/gimli/elf.rs

```diff
@@ -347,6 +357,13 @@ fn decompress_zlib(input: &[u8], output: &mut [u8]) -> Option<()> {
     }
 }
 
+fn decompress_zstd(input: &[u8], output: &mut [u8]) -> Option<()> {
+    use ruzstd::io::Read;
+
+    let mut decoder = ruzstd::StreamingDecoder::new(input).ok()?;
+    decoder.read_exact(output).ok()
```

Comment:
So we're using `read_exact` here: why?

Reply:
Hmm, because it should have the size information from before...

Reply:
The ELF compression header includes the the decompressed size, so we know exactly how many bytes it should decompress to, and we've already allocated the buffer for that. So `read_exact` seems like the correct option to me. The only thing it is missing is an error check to ensure there aren't any more bytes left to read.